locales/en-US/tools.ftl

```diff
@@ -149,8 +153,8 @@ install-other-methods-link = Learn more
 
 ## components/tools/rustup.hbs
 
-tools-rustup-unixy = It looks like you’re running macOS, Linux, or another Unix-like OS. To download Rustup and install Rust, run the following in your terminal, then follow the on-screen instructions.
-tools-rustup-windows = It looks like you’re running Windows. To install Rust, download and run the following, and then follow the onscreen instructions.
+tools-rustup-unixy = It looks like you’re running macOS, Linux, or another Unix-like OS. To download Rustup and install Rust, run the following in your terminal, then follow the on-screen instructions. If you are using Windows, <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">click here</a>
```

Comment:
Don't use "click here" for links, it's bad content. The link text should describe what you're clicking on.
```suggestion
tools-rustup-unixy = It looks like you’re running macOS, Linux, or another Unix-like OS. To download Rustup and install Rust, run the following in your terminal, then follow the on-screen instructions. See <a href="https://forge.rust-lang.org/infra/other-installation-methods.html">"Other Installation Methods"</a> if you are on Windows.
```